app/services/datastore/get_applications.rb

```diff
@@ -0,0 +1,34 @@
+module Datastore
+  class GetApplications
+    def initialize(status: nil, page: nil, per_page: nil, sort: nil)
+      @status = status
+      @page = page
+      @per_page = per_page
+      @sort = sort
+    end
+
+    def call
+      result = DatastoreApi::Requests::ListApplications.new(
+        status: status_filter,
+        page: @page,
+        per_page: @per_page,
+        sort: @sort
+      ).call
+
+      Kaminari.paginate_array(result, total_count: result.pagination['total_count'])
+    rescue StandardError => e
+      Rails.logger.error(e)
+      Sentry.capture_exception(e)
```

Comment:
Does this return a `nil`? The call to capture_exception might be returning any other thing and then the safe navigation `&.page` might trigger inadvertently. I think is best to explicitly add a `nil` as a return value to the rescue.